lightning/src/util/events.rs

```diff
@@ -1180,7 +1194,7 @@ impl MaybeReadable for Event {
 			29u8 => {
 				let f = || {
 					let mut channel_id = [0; 32];
-					let mut user_channel_id: u64 = 0;
+					let mut user_channel_id: u128 = 0;
```

Comment:
Just checking -- we're fine breaking compat here because this event hasn't been included in a release yet?

Reply:
Yup, see https://github.com/lightningdevkit/rust-lightning/pull/1790#discussion_r1016352730.